acoustid/future/fpindex/client.py

```
@@ -0,0 +1,282 @@
+    def _url(self, path: str) -> str:
+        """Build a URL for the given path"""
+        return urljoin(self.base_url, path)
+
+    async def _request(
+        self,
+        method: str,
+        path: str,
+        data: Optional[Dict] = None,
+        timeout: Optional[float] = None,
+        expected_status: Optional[List[int]] = None,
+    ) -> Tuple[int, Any]:
```

Ruff (0.8.2)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)